app-emulation/docker/files/early-docker.service

```diff
@@ -1,13 +1,17 @@
 [Unit]
 Description=Early Docker Application Container Engine
 Documentation=http://docs.docker.com
-After=early-docker.socket
-Requires=early-docker.socket
+After=containerd.service early-docker.socket
+Requires=containerd.service early-docker.socket
```

Comment:
You are using the same containderd for both docker and early-docker. I seem to recall @mischief discovering that this isn't possible. Is that not the case?